scripts/i.tasscap/i.tasscap.html

```diff
@@ -86,7 +86,8 @@ <h2>REFERENCES</h2>
 <li>LANDSAT-7: TASSCAP factors cited from:
   DERIVATION OF A TASSELED CAP TRANSFORMATION BASED ON LANDSAT 7 AT-SATELLITE REFLECTANCE.
   Chengquan Huang, Bruce Wylie, Limin Yang, Collin Homer and Gregory Zylstra Raytheon ITSS,
-  USGS EROS Data Center Sioux Falls, SD 57198, USA (http://landcover.usgs.gov/pdf/tasseled.pdf).
+  USGS EROS Data Center Sioux Falls, SD 57198, USA
+  (<a href="https://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1630&context=usgsstaffpub">PDF</a>).
```

Comment:
DOI is : https://doi.org/10.1080/01431160110106113
https://digitalcommons.unl.edu/usgsstaffpub/621/ is the canonical page linked in the PDF's cover page. The link here is the (current, implementation details) PDF download link when clicking on the dowload link of https://digitalcommons.unl.edu/usgsstaffpub/621/ as of now

Reply:
The DOI leads to a paywall, not really helpful...
So I have chosen https://digitalcommons.unl.edu/usgsstaffpub/621/